js/beerxml-to-beerjson.js

```diff
@@ -1,138 +1,125 @@
 const XML = require('pixl-xml')
-const _ = require('lodash')
+const {
+  isNil,
+  get,
+  map,
+  every,
+  isEmpty,
+  toLower,
+  lowerCase
+} = require('lodash')
 
 const parseBool = s => s === 'TRUE'
-const getArrayNode = node => Array.from(Array.isArray(node) ? node : [node])
+const getArrayNode = node =>
+  Array.from(isNil(node) ? [] : Array.isArray(node) ? node : [node])
 
 const getStyleType = type =>
-  type === 'lager' || type === 'ale' ? 'beer' : type
+  type === 'lager' || type === 'ale' || type === '' ? 'beer' : type
+const getFermentableType = type =>
+  [
+    'dry extract',
+    'extract',
+    'grain',
+    'sugar',
+    'fruit',
+    'juice',
+    'honey',
+    'other'
+  ].indexOf(type) == -1
+    ? 'other'
+    : type
 
 const potential = y => (y * 0.01 * 46) / 1000 + 1
```

Comment:
This could be possibly be reduced to `const potential = y => (y * 0.46) / 1000 + 1;`

Reply:
Yes, initially I used the following conversion formula: Extract potential (S.G.) = 1 + (DBFG / 100) * 0.04621 https://byo.com/article/understanding-malt-spec-sheets-advanced-brewing/